Exiled.API/Features/Npc.cs

```diff
@@ -196,5 +213,125 @@ public void Destroy()
             Dictionary.Remove(GameObject);
             Object.Destroy(GameObject);
         }
+
+        /// <summary>
+        /// Makes the NPC look at the specified position.
+        /// </summary>
+        /// <param name="position">The position to look at.</param>
+        public void LookAt(Vector3 position)
+        {
+            if (RoleManager.CurrentRole is IFpcRole fpc)
+                fpc.LookAtPoint(position);
+        }
+
+        /// <summary>
+        /// Makes the NPC Shoot at the specified location if holding a <see cref="Firearm"></see>.
+        /// </summary>
+        /// <param name="targetPos">The position to shoot towards.</param>
+        public void Shoot(Vector3? targetPos = null)
```

Comment:
Can be converted to book and return false if player is not holding a firearm or not IFpcRole etc